app/src/main/java/ch/epfl/sdp/blindly/settings/Settings.kt

```diff
@@ -34,25 +34,25 @@ class Settings : AppCompatActivity() {
         supportActionBar?.hide()
 
         val emailAddressText = findViewById<TextView>(R.id.email_address_text)
-        emailAddressText.text = user.getEmail() ?: "NOT LOGGED-IN";
+        emailAddressText.text = user.getEmail() ?: getString(R.string.not_logged_in);
 
         val locationText = findViewById<TextView>(R.id.current_location_text)
-        locationText.text = "Lausanne, Switzerland"
+        locationText.text = getString(R.string.lausanne_switzerland)
 
         val radiusText = findViewById<TextView>(R.id.radius_text)
         val radiusSeekBar = findViewById<SeekBar>(R.id.seekBar)
 
         radiusSeekBar.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
             override fun onProgressChanged(seek: SeekBar, progress: Int, fromUser: Boolean) {
-                radiusText.text = progress.toString() + "km"
+                radiusText.text = progress.toString() + getString(R.string.km)

```

Comment:
Maybe follow Cirrus advice here instead of using concatenation ?

Reply:
Done.